framework/test/unit/modules/interoperability/base_cross_chain_update_command.spec.ts

```diff
@@ -753,12 +752,17 @@ describe('BaseCrossChainUpdateCommand', () => {
 				{
 					ccm: { ...EmptyCCM, params: EMPTY_BYTES },
 					result: CCMProcessedResult.DISCARDED,
-					code: CCMProcessedCode.INVALID_CCM_VALIDATION_EXCEPTION,
+					code: CCMProcessedCode.INVALID_CCM_VALIDATION_EXCEPTION, // ***
```

Comment:
we can remove `// ***`

Reply:
I think maybe its better to remove it or have a proper message as currently its only making it confusing